src/mqt/predictor/ml/helper.py

```diff
@@ -28,42 +24,15 @@ def qcompile(
     """Compiles a given quantum circuit to a device with the highest predicted figure of merit.
 
     Arguments:
-        qc: The quantum circuit to be compiled.
+        qc: The quantum circuit to be compiled
```

Comment:
```suggestion
        qc: The quantum circuit to be compiled.
```
Irrelevant change that introduces inconsistency.